packages/components/src/EditableText/EditableText.scss

```diff
@@ -13,10 +13,19 @@ $tc-circle-editable-text-size: 16px !default;
 	@include heartbeat(object-blink);
 }
 
-.tc-editable-text {
-	margin: 0 110px 0 0;
-	max-width: 90rem;
+:global {
```

Comment:
I think this global-block and its content can be removed. It seems to be duplicating the styles further down:
```
			&-submit {
				&:focus,
				&:hover {
					color: $scooter;
					background: $scooter;
					border-color: $scooter;
					& svg {
						background: $scooter;
					}
				}
```

Reply:
ty for spotting it, it's now deleted